spec/models/user_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:surname) }
+  it { should validate_presence_of(:city) }
```

Comment:
I would update the syntax to `it { is_expected.to validate_presence_of(:name) }`